docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+
+**Responsibilities:**
+* Providing updates to the mailing list with the progress of the release
+* Coordinating directly with the WG leads about dates and deliverables
+* Coordinating with the Release Team Members for the progress of the release
+* Ensuring the necessary github branches and tags are cut for the different phases of the release
+* Driving the Release Team meetings
+* All the responsibilities of a Release Team Member
+
+### Release Team Member
+Release Team Members ensure that there's enough bandwidth to perform release relative tasks. They work close with the Release Manager to address release issues.
+
+**Prerequisites:** Have contributions in at least 2 Working Groups
+
+**Time Commitments:** Their role will need to be synchronous, since they are expected to join the Release meetings. Their time commitment is expected to be spread out throughout the release.
+
+**Responsibilities:**
+
+* Keeping track of the provided component versions and updating the [versions table](https://github.com/kubeflow/manifests#kubeflow-components-versions)
+* Creating PRs to update manifests repo from WG provided git revisions
+* Attending the Release Team meetings
+* Providing feedback on enhancing the current Release Handbook
+
+### Product Manager
+They will be in charge of driving non-code deliverables, like the post-release blog post and social media announcements. The Product Manager is also responsible for tracking important features and bug fixes that should be highlighted in the release.
+
+**Prerequisites:** Experience in developing blog posts. Contributions to an existing Open Source project is a huge plus.
+
+**Time Commitments:** This role will require synchronous communication, since they need to be attending the Release Team meetings. Most of the effort is expected to start once the Feature Freeze phase completes.
+
+**Responsibilities:**
+
+* Driving the blog post effort
+* Keeping notes from the Release Team meetings
+* Making sure the processes are being followed
+* Handling communication for social media content publication
+
+### Docs Lead
+The Docs Lead is responsible for working with the Release Team to coordinate documentation updates for the next Kubeflow release.
+
+**Prerequisites:** Contributions to the [website repo](https://github.com/kubeflow/website)
+
+**Time Commitments:** This role will require some synchronous communication. The bulk of their work is expected to be in the Documentation phase of the release, but they will also be involved throughout the release by keeping track of issues that might require an update to the docs.
+
+**Responsibilities:**
+* Identifying and tracking new issues that require update to the docs
+* Working with contributors to modify existing docs to accurately represent any upcoming changes
+* Reviewing documentation PRs to ensure quality following the website [Style Guide](https://www.kubeflow.org/docs/about/style-guide/)
```

Comment:
This means the Docs lead also likely needs to be an APPROVER for the website.

Reply:
I think it should be fine for now to keep the minimum requirements, which are the Docs Lead to review the PRs.

But if we see that there's a blocker from the approvers of the website repo then we could evaluate again if it would make sense to explicitly ask for more permissions